google-cloud-testing/storage-benchwrapper/src/main/java/com/google/cloud/benchwrapper/Main.java

```diff
@@ -30,13 +30,13 @@ public static void main(String[] args) throws Exception {
     }
 
     String storageEmulatorHost = System.getenv("STORAGE_EMULATOR_HOST");
-    if (storageEmulatorHost == null || storageEmulatorHost.equals("")) {
-      // We could use system properties here too, but every other language uses
-      // an environment variable called STORAGE_EMULATOR_HOST, so the
-      // consistency is nice to maintain.
-      System.err.println("Please set STORAGE_EMULATOR_HOST=localhost:8080");
-      System.exit(1);
-    }
+    // if (storageEmulatorHost == null || storageEmulatorHost.equals("")) {
```

Comment:
Do you want to keep in the mean time? Otherwise it could be better to delete if not necessary.

Reply:
I can delete it. since this is in version control anyway.